client/rest_client.py

```diff
@@ -0,0 +1,264 @@
+import time
+
+import requests
+import clkhash
+from clkhash.backports import TimeoutError
```

Comment:
we don't need that any more as well

Reply:
Why don't we need TimeoutError?

Reply:
we don't need the one from backports. python3 comes with TimeoutError as standard